pom.xml

```diff
@@ -91,27 +109,108 @@
                 <testng.suite.file>${testng.configs.path}/test.all.xml</testng.suite.file>
             </properties>
         </profile>
+        <profile>
+            <id>github-page-test-ex1-ex2-tests</id>
+            <properties>
+                <testng.suite.file>${testng.configs.path}/test.GithubPageSeleniumEx1Ex2</testng.suite.file>
+            </properties>
+        </profile>
+        <profile>
+            <id>page-object-tests</id>
+            <properties>
+                <testng.suite.file>${testng.configs.path}/test.PageObject.xml</testng.suite.file>
+            </properties>
+        </profile>
+        <profile>
+            <id>page-object-allure-tests</id>
+            <properties>
+                <testng.suite.file>${testng.configs.path}/test.PageObjectAllure.xml</testng.suite.file>
+            </properties>
+        </profile>
+        <profile>
+            <id>page-object-selenium-grid-tests</id>
+            <properties>
+                <testng.suite.file>${testng.configs.path}/test.PageObjectSeleniumGrid.xml</testng.suite.file>
+            </properties>
+        </profile>
+        <profile>
+            <id>jdi-tests</id>
+            <activation>
+                <activeByDefault>true</activeByDefault>
+            </activation>
+            <properties>
+                <testng.suite.file>${testng.configs.path}/test.jdi.xml</testng.suite.file>
+            </properties>
+        </profile>
     </profiles>
 
     <dependencies>
-        <dependency>
-            <groupId>org.testng</groupId>
-            <artifactId>testng</artifactId>
-            <version>${testng.version}</version>
-            <scope>test</scope>
-        </dependency>
         <dependency>
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
             <version>${assertj-core.version}</version>
             <scope>test</scope>
         </dependency>
+        <!--<dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-simple</artifactId>
+            <version>${slf4j-simple.version}</version>
+        </dependency>-->
         <dependency>
             <groupId>calculator</groupId>
             <artifactId>calculatorapp</artifactId>
             <scope>system</scope>
             <version>1.0</version>
             <systemPath>${project.basedir}/src/main/resources/calculator-1.0.jar</systemPath>
         </dependency>
+        <!--<dependency>
```

Comment:
It's not a good practice to comment code. Just remove if you do not need it. It's very easy to recover any code from version control